RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@ Unreleased
 
 1.52.1
 ------
+* [**] Added transform block capability [#3321]
```

Comment:
I thought this was in 1.52.0. Was this in the hotfix because of the bug where it was left disabled on debug builds? I thought that bug was on the native side and had nothing to do with gutenberg-mobile. I may just be misremembering though. 🤔

Reply:
Actually you're completely right, it was in 1.52.0. I just saw the 52 and somehow didn't notice the 1.